tldw_Server_API/app/core/DB_Management/Collections_DB.py

```
@@ -1551,6 +1557,36 @@ def ensure_schema(self) -> None:
                     logger.debug("collections backfill: content_items.{} already exists or skipped", column)
                 else:
                     raise
+        # Backfill user_notifications columns
+        notif_columns: set[str] = set()
+        if self.backend.backend_type == BackendType.SQLITE:
+            try:
+                notif_columns = self._sqlite_columns("user_notifications")
+            except _COLLECTIONS_NONCRITICAL_EXCEPTIONS:
+                pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(_COLLECTIONS_NONCRITICAL_EXCEPTIONS)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(_COLLECTIONS_NONCRITICAL_EXCEPTIONS): ...`

(SIM105)